src/main/java/ai/elimu/tasks/LetterSoundUsageCountScheduler.java

```diff
@@ -0,0 +1,69 @@
+package ai.elimu.tasks;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.logging.log4j.Logger;
+import ai.elimu.dao.WordDao;
+import ai.elimu.model.content.Sound;
+import ai.elimu.model.content.Letter;
+import ai.elimu.model.content.LetterSound;
+import ai.elimu.model.content.Word;
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import ai.elimu.dao.LetterSoundDao;
+
+@Service
+public class LetterSoundUsageCountScheduler {
+    
+    private Logger logger = LogManager.getLogger();
+    
+    @Autowired
+    private WordDao wordDao;
+
+    @Autowired
+    private LetterSoundDao letterSoundDao;
+    
+    @Scheduled(cron="00 15 06 * * *") // At 06:15 every day
+    public synchronized void execute() {
+        logger.info("execute");
+        
+        logger.info("Calculating usage count for LetterSounds");
+
+        // <id, usageCount>
+        Map<Long, Integer> letterSoundFrequencyMap = new HashMap<>();
+
+        List<Word> words = wordDao.readAll();
+        logger.info("words.size(): " + words.size());
+        for (Word word : words) {
+            logger.info("word.getText(): " + word.getText());
```

Comment:
@SnehaHS65 This file is not in sync with the most recent changes on the `main` branch: https://github.com/elimu-ai/webapp/pull/1732/files#diff-82052f8d582b9a0720ed15bc313d8c8fa17f12e2cf7ec8677aa3ebe828e22a24

Reply:
@SnehaHS65 Reminder to not change from `logger.debug` to `logger.info`.